src/osw_incline/osm_graph.py

```diff
@@ -82,14 +99,18 @@ def to_geojson(self, *args):
                     'geometry': geometry,
                     'properties': d_copy
                 })
-        nodes_fc = {'type': 'FeatureCollection', 'features': node_features}
```

Comment:
simply change this to
`node_fc = {'type':'FeatureCollection', '$schema':"https://sidewalks.washington.edu/opensidewalks/0.2/schema.json", 'features': node_features}`